spec/models/podcast_spec.rb

```diff
@@ -4,6 +4,6 @@
   it { should have_valid(:name).when("Podcast1") }
   it { should_not have_valid(:name).when(nil, "") }
 
-  it { should have_valid(:url).when("www.podcast2.com") }
-  it { should_not have_valid(:url).when(nil, "") }
-end
+  it { should have_valid(:url).when("http://www.podcast2.com") }
```

Comment:
could also include "https" as a valid example because you check it when validating: `.when("http://www.podcast2.com", "https://www.podcast2.com")`